bb-pilcom/bb-pil-backend/templates/prover.cpp.hbs

```diff
@@ -93,12 +93,13 @@ void {{name}}Prover::execute_log_derivative_inverse_commitments_round()
 {
     // Commit to all logderivative inverse polynomials
     for (auto [commitment, key_poly] : zip_view(witness_commitments.get_derived(), key->get_derived())) {
-       commitment = commitment_key->commit(key_poly);
+        // We don't use commit_sparse here because the logderivative inverse polynomials are dense
+        commitment = commitment_key->commit(key_poly);
```

Comment:
It depends some lookups are, as they are not active on every row

Reply:
Grand product polys for the copy constraints are dense though so we can treat those differently

Reply:
Ah good point. I'll merge this one but we can consider later.

I did try with `commit_sparse` here and did see an improvement of some % points, but not as big as the other case. I preferred to err on the side of caution mem-wise, since the current implementation copies the poly.